### PR TITLE
Placeholder

### PR DESCRIPTION
**Pull request by chadkoh**
This just builds on the basic info right now with some (okay) copy and links to all our properties. Still no header image for now.

### CODE REVIEW

**Comment by chadkoh** [comment]
pulled down and merged current master, cleaning up the conflicts. Should be good to go!

File: index.html [hunk 1]
```diff
@@ -25,150 +25,225 @@
...
-      <div class="ui container">
-        <a class="active item">Home</a>
-        <a class="item">Work</a>
-        <a class="item">Company</a>
-        <a class="item">Careers</a>
-      </div>
-    </div>
+    <style type="text/css">
+
+      .hidden.menu {
```
Comment (jakeorr) [inline index.html]: If any of these aren't in the [custom stylesheet](https://github.com/jakeorr/oktabletop/blob/master/public/styles/style.css) can you move them in there? Let's leave all our styles in there, this style block can be removed.

**Comment by jakeorr** [comment]
This is looking good! I love the quotes. That "Get Started" button in the header doesn't do anything. Do you want it in there?

**Comment by chadkoh** [comment]
removed.

**Pull request merged**